test/settings.py

```diff
@@ -1,13 +1,13 @@
-import os
-import sys
+import os.path
```

Comment:
`os.environ` is also used in this module!

Reply:
actually `os.path` module even import `os` module behind the scenes. both of them different modules, so to be more explicit I think we can import both.